contract_reader.py

```diff
@@ -178,6 +183,8 @@ def get_account_balances(tokens: List[str],
 
     """
     # Get token adresses first.
+    contract = init_contract(
+        network)
```

Comment:
This formatting looks a little weird to me. Seems like your linter is very strict :-)